src/angularComponent/ngcOmniboxController.js

```diff
@@ -178,11 +178,23 @@ export default class NgcOmniboxController {
         this.suggestions = Array.prototype.slice.apply(suggestions);
         this._buildSuggestionsUiList();
       } else {
-        this.suggestions = suggestions;
+        throw new Error('Suggestions must be an Array');
       }
     });
   }
 
+  set suggestions(suggestions) {
```

Comment:
Doesn't this change also affect a unit test file? I saw you manually calling `_buildSuggestionsUiList()` in one of them.